src/core/uploadFrameMediaToIPFS.ts

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs';
```

Comment:
Are we looking to only support this to `node`?

Reply:
What runtimes does OnchainKit support? I can try to make this general purpose.